src/editor-service/editor-server.ts

```diff
@@ -97,7 +97,7 @@ process.stdin.pipe(split()).on('data', async function(line: string) {
 // node child_process.fork() IPC interface
 process.on('message', async function(request: RequestWrapper) {
   const result = await getSettledValue(request.value);
-  process.send(<ResponseWrapper>{id: request.id, value: result});
+  process.send!(<ResponseWrapper>{id: request.id, value: result});
```

Comment:
It's a cast. If you have `o: T|null`, `o!` is the same as `(o as T)`. If TypeScript had subtraction types, it'd be `(o as (typeof o - null|undefined))`.